fuse/moosefs/Portfile

```diff
@@ -0,0 +1,109 @@
+                    port:zlib
+ 
+depends_build       port:pkgconfig
+
+default_variants    +servers +client
+set mfs_cliconfs    {mfsmount.cfg}
+set mfs_servconfs   {mfschunkserver.cfg mfsexports.cfg mfshdd.cfg mfsmaster.cfg mfsmetalogger.cfg mfstopology.cfg}
+set mfs_masterfiles {metadata.mfs}
+configure.args      --disable-all
+notes               ""
+
+livecheck.type      regex
+livecheck.url       http://ppa.moosefs.com/moosefs-3/version
+livecheck.regex     {^(.*)$}
+
+variant servers description {All server-side needed binaries} {
+    add_users               _mfs group=_mfs realname=MooseFS\ User home=${prefix}/var/mfs
+    configure.args-append   --enable-mfscgi \
+                            --enable-mfscgiserv \
+                            --enable-mfschunkserver \
+                            --enable-mfscli \
+                            --enable-mfsmaster \
+                            --enable-mfsmetalogger \
+                            --enable-mfsnetdump \
+                            --with-default-user=_mfs \
+                            --with-default-group=_mfs
+    startupitem.create      yes
+    startupitems            name mfsmaster \
+                            location LaunchDaemons \
+                            start "${prefix}/sbin/mfsmaster start" \
+                            stop "${prefix}/sbin/mfsmaster stop" \
+                            restart "${prefix}/sbin/mfsmaster restart" \
+                            name mfschunkserver \
+                            location LaunchDaemons \
+                            start "${prefix}/sbin/mfschunkserver start" \
+                            stop "${prefix}/sbin/mfschunkserver stop" \
+                            restart "${prefix}/sbin/mfschunkserver restart" \
+                            name mfsmetalogger \
+                            location LaunchDaemons \
+                            start "${prefix}/sbin/mfsmetalogger start" \
+                            stop "${prefix}/sbin/mfsmetalogger stop" \
+                            restart "${prefix}/sbin/mfsmetalogger restart"
+    notes-append            "
+You have just installed MooseFS Master, Chunkserver and Metalogger binaries. To run them automatically during system startup,\
+you need to activate them. Use these commands:
+sudo launchctl load -w /Library/LaunchDaemons/org.macports.mfsmaster.plist
+sudo launchctl load -w /Library/LaunchDaemons/org.macports.mfschunkserver.plist
+sudo launchctl load -w /Library/LaunchDaemons/org.macports.mfsmetalogger.plist
+to activate Master, Chunkserver and Metalogger daemons respectively. The command suggested to you by macports will activate\
+ALL daemons at the same time, so don't use it if you'd rather activate just one of them.
+Chunkserver and Metalogger will not run if they cannot resolve the hostname 'mfsmaster' (unless you changed the default\
+setting for MASTER_HOST in mfschunkserver.cfg or mfsmetalogger.cfg respectively).
+=========================
```

Comment:
Please remove this line.

Reply:
I assumme you mean the "====" line. How else should I separate variants' comments from each other? Will a blank line be displayed properly?